media-libs/jpeg/jpeg-9a.recipe

```diff
@@ -65,6 +65,9 @@ BUILD_PREREQUIRES="
 	cmd:make
 	"
 
+defineDebugInfoPackage jpeg \
+	$libDir/libjpeg.so.9.1.0
```

Comment:
It's a pity to have the version hardcoded here. $libDir/libjpeg.so.*.*.* should do the same, shouldn't it?